tracer/src/Datadog.Trace/AppSec/Waf/Context.cs

```diff
@@ -42,23 +44,22 @@ public IResult Run(IDictionary<string, object> args, ulong timeoutMicroSeconds)
                 Log.Debug("DDAS-0010-00: Executing AppSec In-App WAF with parameters: {Parameters}", parameters);
             }
 
-            var rawAgs = pwArgs.RawPtr;
+            var rawArgs = pwArgs.RawPtr;
             DdwafResultStruct retNative = default;
-
-            var code = wafNative.Run(contextHandle, rawAgs, ref retNative, timeoutMicroSeconds);
-
-            var ret = new Result(retNative, code, wafNative);
+            var code = wafNative.Run(contextHandle, rawArgs, ref retNative, timeoutMicroSeconds);
+            stopwatch.Stop();
+            _totalRuntimeOverRuns += retNative.TotalRuntime / 1000;
+            var result = new Result(retNative, code, wafNative, _totalRuntimeOverRuns, (ulong)stopwatch.Elapsed.TotalMilliseconds * 1000);
```

Comment:
I noticed that the last parameter for the constructor feeds into the `AggregatedTotalRuntimeWithBindings` property. Do you also need to store a `private ulong` field so you can accumulate the stopwatch durations over time?

Reply:
Context is request scoped, so the stopwatch is the same across the request, and starting it is resuming it in case it had been already started before, so the stopwatch is already giving us the accumulated timings as I checked (?)

Reply:
Oh I see, that was my mistake I forgot that the stopwatch continued to accumulate between requests. Sorry about that 😅